src/double-commas.js

```diff
@@ -28,7 +28,7 @@
 
 'use strict';
 
-export default async function() {
+export default async function () {
```

Comment:
Stash changes to this file. The PR should not modify this validator.

Reply:
Fixed